rgw/admin/user.go

```diff
@@ -33,6 +33,7 @@ type User struct {
 	PurgeData           *int          `url:"purge-data"`
 	GenerateStat        *bool         `url:"stats"`
 	Stat                UserStat      `json:"stats"`
+	UserCaps            string        `url:"user-caps"`
```

Comment:
Just a minor nitpick, one that you probably have no control over: it seems a little strange that we have a Caps field with type `[]UserCapSpec` and now add a UserCaps field with type `string`. However, since I already see that the json field names for these are "caps" and "user-caps" I can only assume this is a quirk of the rgw admin api.
If this is the case I think this is fine to match.  I just wanted to double check on that first before approving, thanks!

Reply:
Yaa it is coming from RGW adminOps rest api [implementation](https://docs.ceph.com/en/latest/radosgw/adminops/). It was there atleast from Nautilus onwards